src/cli/add.rs

```diff
@@ -129,6 +129,18 @@ pub async fn execute(args: Args) -> miette::Result<()> {
                 .collect();
 
             if let Some(git) = &dependency_config.git {
+                if !workspace
+                    .manifest()
+                    .workspace
+                    .preview()
+                    .is_enabled(KnownPreviewFeature::PixiBuild)
+                {
+                    return Err(miette::miette!(
+                        help = "Add `workspace.preview = [\"pixi-build\"]` to enable pixi build support",
+                        "source dependencies are not allowed without enabling pixi-build"
```

Comment:
```suggestion
                        help = "Add `preview = [\"pixi-build\"]` to the `workspace` or `project` table of your manifest",
                        "conda source dependencies are not allowed without enabling the 'pixi-build' preview feature"
```
Would be nice to replace the word `manifest` with the actual manifest path to avoid confusion where to add it.